examples/nuxt-openai/server/api/assistant.ts

```diff
@@ -0,0 +1,118 @@
+import { AssistantResponse } from 'ai';
+import OpenAI from 'openai';
+
+const homeTemperatures = {
+  bedroom: 20,
+  'home office': 21,
+  'living room': 21,
+  kitchen: 22,
+  bathroom: 23,
+};
+
+export default defineEventHandler(async event => {
+  const config = useRuntimeConfig(event);
+
+  if (!config.openaiApiKey) {
+    throw new Error('OPENAI_API_KEY is not set');
```

Comment:
Should we have NUXT_ prefix in these logs?

Reply:
Yes